app/lib/actions/katello/pulp_selector.rb

```diff
@@ -14,7 +14,11 @@ def plan_optional_pulp_action(backend_actions, repository, smart_proxy, *args)
       end
 
       def select_pulp_action(backend_actions, repository, smart_proxy)
-        backend_type = smart_proxy.backend_service_type(repository)
+        if smart_proxy.pulp2_preferred_for_type?(repository.content_type.to_sym)
+          backend_type = "pulp2"
+        else
+          backend_type = smart_proxy.backend_service_type(repository)
+        end
```

Comment:
since backend_service_type calls pulp3_support?, i don't think we need this particular check